04 - Guides, Workflows, & Courses/for Knowledge Management.md

```diff
@@ -8,6 +8,8 @@ tags:
 
 # for Knowledge Management
 
+- [Universal knowledge management rules](https://davidewiest.medium.com/universal-pkm-rules-ddbea35bea3b)
```

Comment:
Many thanks for this.

I'm happy to merge it is as-is, but for tidiness, as all the other links are in headings, can you think of a heading to use for the new link - or does it fit in one of the existing headings?

Reply:
Ok, I came up with the heading "Fundamental information".